### ISSUE TITLE
Create operation always enables projection, but it should not

### ISSUE THREAD

**Issue by michael-schnell** [EventStore/EventStoreDB-Client-Java#259]
Currently there seems to be no way to create a **disabled** projection using the **EventStoreDBProjectionManagementClient**.

The documentation of the **create** operation states:
> Creates a new projection in the stopped state. Enable needs to be called separately to start the projection.

But when the following code is executed, the projection is already enabled (even if "enable" was not called afterwards):
```java
    public static void main(String... args) throws ExecutionException, InterruptedException {

        EventStoreDBProjectionManagementClient client = EventStoreDBProjectionManagementClient
                .create(EventStoreDBConnectionString
                        .parseOrThrow("esdb://localhost:2113?tls=false"));

        String javascript = "fromAll().foreachStream().when({'one': function(state, ev) { linkTo('test-delete', ev); },'two': function(state, ev) { linkTo('test-delete', ev); }})";

        client.create("my-projection", javascript,
                CreateProjectionOptions.get().emitEnabled(false).trackEmittedStreams(true)).get();

    }
```

This means there is currently no way to create a disabled projection using the API.

Full Maven project for testing can be found here: https://github.com/michael-schnell/eventstore-create-enabled-bug

<sub>[DEV-229](https://linear.app/eventstore/issue/DEV-229/create-operation-always-enables-projection-but-it-should-not)</sub>

**Comment by w1am** [comment]
Right now, you can't do that with the gRPC client. But it will be amended.

**Comment by michael-schnell** [comment]
Any update here?

**Comment by YoEight** [comment]
Let me rephrase what @w1am mentioned. The client uses the gRPC protocol, which, as of now, doesn’t provide a way to disable a projection on creation. This isn’t a Java-specific client — it’s simply a gRPC-based one. Currently, the only API that supports it is the projection REST API, which the Java client does not use.

We have no plans to support a hybrid client, especially for the projections API. However, we do intend to release a dedicated tool that will cover this need, including support for the user management API. Unfortunately, I don't have an ETA for that.

In your case, using the projection REST API directly would be the better approach.
